### PR TITLE
Update hugo-bin to v0.43.1 (Hugo 0.55.1).

### PR DESCRIPTION
**Pull request by XhmikosR**
Requires #28570.

### CODE REVIEW

File: site/layouts/_default/examples.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="description" content="">
     <meta name="author" content="{{ .Site.Params.authors }}">
-    <meta name="generator" content="Hugo {{ .Hugo.Version }}">
+    <meta name="generator" content="Hugo {{ .Site.Hugo.Version }}">
```
Comment (XhmikosR) [inline site/layouts/_default/examples.html]: Not sure if this 100% right even though it works. Maybe `{{ hugo.Version }}` is the proper way @bep?

Comment (bep) [reply site/layouts/_default/examples.html]: Yes.

**Comment by coliff** [comment]
Hi @XhmikosR - maybe in this PR you could add:
`{{ printf "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"yes\" ?>" | safeHTML }}` to the top of the `sitemap.xml` in the layouts dir. Might already be in your todo list but thought I'd mention. :-) 

(I just updated a bunch of sites to hugo v0.5.5 and had to do that as mentioned in the Hugo blog Notes section https://gohugo.io/news/0.55.0-relnotes/ )

**Comment by XhmikosR** [comment]
@coliff: thanks, I definitely missed this. I will update this branch soon.

**Review by Johann-S — APPROVED** [review]

**Pull request merged**